main.go

```diff
@@ -115,6 +115,7 @@ func main() {
 		},
 		KubeRay: &config.KubeRayConfiguration{
 			RayDashboardOAuthEnabled: pointer.Bool(true),
+			IngressDomain:            "kind",
```

Comment:
Is there a specific reason to set kind domain here?

Reply:
You have an eagle eye, thanks for pointing that out. That was done for my own testing, changed back to `fake.domain` (or whichever other default domain name we prefer). Thanks!